armv7hf-buildpack-deps/README-short.txt

```diff
@@ -1 +1 @@
-A collection of common build dependencies used for installing various modules, e.g., gems.
+This repository is obsolete.
```

Comment:
@Page- README-short is the content for the headline. Link to our docker image document is included in the full description.